services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationService.kt

```diff
@@ -197,7 +197,7 @@ open class ApplicationService(
         val application = getById(id)
 
         val hanke = application.hanke
-        val hankeId = hanke.id!!
+        val hankeId = hanke.id
```

Comment:
This can also possibly be removed.

Reply:
Thank you for noticing. I inlined the value.